meta/tasks.py

```diff
@@ -1,18 +1,27 @@
+import sys
 from pathlib import Path
 
-from devutils.invoke_utils import build_common_tasks
 from invoke import task
 
-CURDIR = Path(__file__).absolute().parent
+ROOT = Path(__file__).resolve().parent
```

Comment:
Don't `resolve()`. Use `absolute()` instead.

Reply:
Maybe that's why I was encountering some issues during Windows development which was accessing files on a mounted drive enabling access to files on my Mac, thing which was messing a little with my drive letters. Although not sure if `absolute()` will still break it.

Was thinking to use `resolve()` just in case we end up with `..` (parent) or `~` (home) in the path somehow.

Reply:
> Maybe that's why I was encountering some issues during Windows development which was accessing files on a mounted drive enabling access to files on my Mac, thing which was messing a little with my drive letters. Although not sure if absolute() will still break it.

Definitely (I do have many scars on that -- `resolve()` is very seldomly needed and is usually the source of bugs). `absolute()` will do the `..` -- for `~` you need `expanduser()` (but in this case we can leave `~` if it ever gets there as there's no issue in that, the `absolute()` is only needed because if `.` was there then we could have a problem).

Reply:
Looks like a strong opinion based on data, so +1 on this!